README.md

```diff
@@ -80,6 +83,21 @@ To Update the module:
 Update-Module -Name WSLBlobNFS
 ```
 
+## Help
+
+1. Use -Verbose switch to get verbose output for the cmdlets.
+
+2. Common issues and their solutions:  
+
+    - If you get the following error while running the Mount-WSLBlobNFS cmdlet, then run ```Initialize-WSLBlobNFS -Force``` and try mounting again :  
+
+        ```powershell
+        Mounting SMB share \\<wsl-ip>\<samba-share-name> onto drive A:
```

Comment:
nit, use Z: here in the example since that's what mostly people will get

Reply:
Added more details on resolving the issue by updating the module and running initialize again.